electrum_nmc/electrum/names.py

```diff
@@ -393,15 +393,26 @@ def get_wallet_name_delta(wallet, tx, domain=None):
     name_input_value = None
     name_output_value = None
 
+    def nameop_is_mine(inout):
+        if inout.address is None:
+          return False
+        if not wallet.is_mine(inout.address):
+          return False
+        if inout.address not in domain:
+          return False
+        if inout.name_op is None:
+          return False
+        return True
```

Comment:
Any reason for making this function part of `get_wallet_name_delta` rather than a separate function?  Also the 2-space indent is inconsistent with the 4-space indent that the rest of the code has.

Reply:
Good catch about the indent, I've fixed that.  The reason for making the function inline is just because it helps keep stuff encapsulated and close together, as it is really just for sharing that logic in two places inside the function, and not something that's needed elsewhere.

If you think you'd rather have it as a separate function, I can of course do that as well.